src/crt.c

```diff
@@ -22,7 +22,7 @@ struct aws_allocator *init_allocator(void) {
     enum aws_mem_trace_level trace_level = AWS_MEMTRACE_NONE;
     struct aws_string *mem_tracing_key = aws_string_new_from_c_str(s_crt_allocator, "AWS_CRT_MEMORY_TRACING");
     struct aws_string *mem_tracing_value = NULL;
-    if (!aws_get_environment_value(s_crt_allocator, mem_tracing_key, &mem_tracing_value)) {
+    if (aws_get_environment_value(s_crt_allocator, mem_tracing_key, &mem_tracing_value)) {
```

Comment:
This will also be wrong, this needs to check that the return value is AWS_OP_SUCCESS and whether the mem_tracing_value is non-null

Reply:
fixed